api/config/config_prod.json

```diff
@@ -101,7 +101,7 @@
     "enableBetaAccess": false,
     "unsafeAllowSelfBypass": false,
     "unsafeAllowUserCreationFromGSuiteData": false,
-    "enableAccessRenewal": false
+    "enableAccessRenewal": true
```

Comment:
As a follow-up we should reach an agreed end-state for this config value in perf and preprod. They are currently `false`. I suspect perf should be `true`, with the same expiry date as test, and preprod should probably just be true with a normal expiry date.